system/ee/ExpressionEngine/View/addons/index.php

```diff
@@ -1,348 +1,290 @@
-                    <div class="dropdown__scroll">
-					            <a class="dropdown__link" href="">Installed</a>
-                      <a class="dropdown__link" href="">Uninstalled</a>
-                      <a class="dropdown__link" href="">Update Available</a>
-            				</div>
-                	</div>
-              	</div>
-        			</div>
-    					<div class="filter-bar__item filter-search-form">
-        				<div class="filter-search-bar__item">
-                	<div class="search-input">
-                		<input class="search-input__input input--small" type="text" name="filter_by_keyword" value="" placeholder="Search">
-                	</div>
-                </div>
-        			</div>
-    					<div class="filter-bar__item ">
-        				<div class="filter-search-bar__item">
-                	<button type="button" class="has-sub filter-bar__button js-dropdown-toggle button button--default button--small" data-filter-label="show">show				<span class="faded">(25)</span>
-            			</button>
-                	<div class="dropdown">
-            				<div class="dropdown__search">
-                			<div class="search-input">
-                  			<input type="text" name="perpage" value="" placeholder="custom limit" data-threshold="1000" data-threshold-text="Viewing more than 1000 items at a time may result in reduced performance." class="search-input__input input--small">
-                			</div>
-                		</div>
-										<a class="dropdown__link" href="">25 results</a>
-										<a class="dropdown__link" href="">50 results</a>
-										<a class="dropdown__link" href="">75 results</a>
-										<a class="dropdown__link" href="">100 results</a>
-										<a class="dropdown__link" href="">150 results</a>
-										<a class="dropdown__link" href="">All Add-Ons</a>
-      						</div>
-                </div>
-        			</div>
-          	</div>
-          </div>
-        </div>
-      </div>
-      <div class="table-responsive table-responsive--collapsible">
-        <table cellspacing="0">
-  				<thead>
-      			<tr class="app-listing__row app-listing__row--head">
-							<th class="column-sort-header column-sort-header--active">
-								<a href="" class="column-sort column-sort--desc">Name</a>
+	<div class="tbl-ctrls">
+		<div class="panel-heading">
+			<div class="app-notice-wrap"><?= ee('CP/Alert')->getAllInlines() ?></div>
+			<?php if (!empty($updates) && count($updates) == 1) : ?>
+				<div id="alert_banner" class="alert">
+					<div class="alert__icon"><i class="fas fa-info-circle fa-fw"></i></div>
+					<div id="alert" href="" class="alert__close">
+						<a id="alert_close" class="fas fa-times alert__close-icon"></a>
+					</div>
+					<p class="alert__title"> <br>
+					<p class="alert__title"><?php echo lang('update'); ?><br>
+						<?php echo lang('there_is'); ?> <strong><?= count($updates) ?></strong> <?php echo lang('single_update'); ?> <a id="ShowMe"><?php echo lang('show'); ?></a></p>
```

Comment:
It might better idea to use single lang key and `sprintf` to insert correct number. This generally will work better through different languages that might have different order of words.

Reply:
In `id="ShowMe"` you are using camel case. I believe we are generally using snake case for HTML attributes. See also https://github.com/ExpressionEngine/ExpressionEngine/blob/6.dev/cp-styles/README.md#expressionengines-control-panel-styles